pages/docs/tutorials/studio-document-validation.md

```diff
@@ -0,0 +1,139 @@
+
+## Copy invalid AsyncAPI document
+Let's pretend we have an invalid AsyncAPI document.
+
+1. Open [Studio](https://studio.asyncapi.com/).
+
+<Remember>
+
+You can also skip the step below by clicking on New File in Studio and opening the Invalid Example template in the tutorials section.
+
+</Remember>
+
+2. Copy and paste the below invalid AsyncAPI document:
+
+```yaml
+asyncapi: 1.0.0
+info:
+  title: Streetlights API
+  version: '1.0.0'
+  description: |
+    The Smartylighting Streetlights API allows you
+    to remotely manage the city lights.
+  license:
+    name: Apache 2.0
+    url: 'https://www.apache.org/licenses/LICENSE-2.0'
+
+servers:
+  mosquitto:
+    host: test.mosquitto.org
+    protocol: mqtt
+
+channels:
+  lightMeasured:
+    address: 'light/measured'
+    messages:
+      lightMeasuredMessage:
+        name: LightMeasured
+        payload:
+          type: object
+          properties:
+            id:
+              type: integer
+              minimum: 0
+              description: Id of the streetlight.
+            lumens:
+              type: integer
+              minimum: 0
+              description: Light intensity measured in lumens.
+            sentAt:
+              type: string
+              format: date-time
+              description: Date and time when the message was sent.
+
+operations:
+  onLightMeasured:
+    action: 'receive'
+    summary: Inform about environmental lighting conditions for a particular streetlight.
+    operationId: onLightMeasured
+    channel:
+      $ref: '#/channels/lightMeasured'
+``` 
+ 
+## Troubleshoot Studio errors
+Let's fix the errors one by one until we end up with a valid AsyncAPI document.
+ 
+1. You can see the error message on the screen: `Empty or invalid document. Please fix errors/define AsyncAPI document.`
+
+2. Open diagnostics, you can see more information related to your errors.
+
+3. Fix the incorrect AsyncAPI specification number to `3.0.0`.
+ 
+```yaml
+asyncapi: '3.0.0'
+info:
+  title: Account Service
+  version: 1.0.0
+```
+<Remember>
+Notice how <b>description</b> property is missing; that doesn't make the AsyncAPI document invalid, but it's always better to include.
+</Remember>
+
+4. Read the next error: `must be number`. Fix the `minimum` by changing it to: `0`.
+
+```yaml
+          properties:
+            id:
+              type: integer
+              minimum: 0
+``` 
+5. You see three errors:
+- must be equal to one of the allowed values
+- must be array
+- must match a schema in `anyOf`
+
+`anyOf` means it should match any one the above schemas then it is valid.
+
+Now let's fix this error by changing the type to `string`
+
+```yaml
+            sentAt:
+              type: string
+              format: date-time
+              description: Date and time when the message was sent.
+```
+
+6. Congratulations! You identified and fixed all the errors, and now have a valid AsyncAPI document.
```

Comment:
```suggestion
5. Congratulations! You identified and fixed all the errors, and now have a valid AsyncAPI document.
```

yeah, I don't think we need to showcase another error with schema, especially that these errors are a bit unclear

Reply:
hmm...this is a bit technical so you would know the best :sweat_smile:

Reply:
resolved with a4beea4